simple-todo/server/src/app.ts

```diff
@@ -10,5 +11,9 @@ const options:CorsOptions = {
 
 app.use(cors(options));
 app.use('/', router);
+app.all('*',async (req,resp,next)=>{
+  throw new Error('test');
```

Comment:
Need to double-check if this is necessary.
Error message should be more descriptive